open-sphere-base/analysis/src/main/java/io/opensphere/analysis/baseball/BaseballController.java

```diff
@@ -69,63 +67,60 @@ private void handleDoubleClickEvent(DataElementDoubleClickedEvent event)
             DataElementLookupUtils lookupUtils = MantleToolboxUtils.getDataElementLookupUtils(myToolbox);
             List<DataElement> dataElements = New.list();
             GeometryFactory geometryFactory = new GeometryFactory();
-            MapGeometrySupport support = lookupUtils.getMapGeometrySupport(event.getRegistryId());
+            GeographicPosition innerPosition = MousePositionUtils.getMousePosition();
```

Comment:
I'm pretty sure it wouldn't be null, since this would only be run after clicking on the map, thus having the mouse position set automatically. I'll add a check in to be thorough.